stumpwm-tests.asd

```diff
@@ -0,0 +1,8 @@
+(defsystem "stumpwm-tests"
```

Comment:
If you don't want to add a separate `.asd` file you can name the system with a slash `stumpwm/tests` and put this in the main `.asd` and ASDF will be smart enough to look there when you try to load it, e.g. https://github.com/sjl/bobbin/blob/master/bobbin.asd

More info in the last paragraph of `find-system` in the ASDF manual: https://common-lisp.net/project/asdf/asdf.html#index-find_002dsystem

Reply:
I'm aware, but I prefer the one file per project. I remember there was a bug with the logic handling the / in ASDF recently (that has already been fixed) where it treated the slash as a sub-directory. I rather make it 'easier' for ASDF.